servers/rendering/renderer_rd/forward_clustered/render_forward_clustered.cpp

```diff
@@ -520,6 +520,10 @@ void RenderForwardClustered::_render_list_template(RenderingDevice::DrawListID p
 			instance_count /= surf->owner->trail_steps;
 		}
 
+		if (primitive == RS::PRIMITIVE_LINES || primitive == RS::PRIMITIVE_LINE_STRIP) {
+			RD::get_singleton()->draw_list_set_line_width(draw_list, 2.0); // To prevent line's flickering.
```

Comment:
This is really only supported on Windows and Linux, it won't do anything on macOS, Android, or iOS. If we want thick lines we probably need to fall back on drawing a screen aligned quad instead :(

Reply:
I wrap it with `#ifdef` to compile only on Windows/Linux. It should be fine then?